rust/helpers.c

```diff
@@ -171,6 +165,24 @@ void rust_helper_rb_link_node(struct rb_node *node, struct rb_node *parent,
 }
 EXPORT_SYMBOL_GPL(rust_helper_rb_link_node);
 
+struct task_struct *rust_helper_current(void)
+{
+	return current;
```

Comment:
Is this a global variable or a macro? Of a global variable, it should be accessible as `static` I think.

Reply:
It's a macro. It resolves to per-CPU locations (given that each CPU has a different current thread/task), similar in concept to userspace thread-local storage.

Reply:
I think the binding should be named `rust_helper_get_current` and bind to `get_current()`. `current` is basically just `#define current get_current()`.

Reply:
> I think the binding should be named `rust_helper_get_current` and bind to `get_current()`. `current` is basically just `#define current get_current()`.

`current` is the idiomatic way to get the current task in C. Do a search for the two and you'll see the difference.

Reply:
I see `current` as a shorthand, and in C that's obvious no reason not to use the shorthand.

However, I think the binding should be named after `get_current` because the binding is a function and it is  used like `let current = rust_helper_get_current();` rather than `let current = rust_helper_current`. The helper could still be implemented with `return current`, I just think the name should change.

If helper generation is made automatic or done like #359, then we should be writing `bindings::get_current()` rather than `bindings::current()`.

Reply:
Ok, I'll change the function name to `rust_helper_get_current()`.